src/test/java/api/items/EffectiveLocationTests.java

```diff
@@ -0,0 +1,75 @@
+package api.items;
+
+import static api.support.matchers.UUIDMatcher.is;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.domain.Item;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.builders.HoldingBuilder;
+import api.support.builders.InstanceBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.http.InventoryItemResource;
+
+public class EffectiveLocationTests extends APITests {
```

Comment:
I'm finding these hard to follow. Are these intended to be unit tests or API tests?

If they are unit tests, please move them to a more appropriate package and remove the dependency on the `APITests` base class.

Reply:
This is actually mix of unit and API tests. There is one API test that creates Item and then verifies that proper locationId returned by getLocationId method + there are few tests that verifies border cases (no item set, no effectiveLocatioId present)

Reply:
> This is actually mix of unit and API tests. There is one API test that creates Item and then verifies that proper locationId returned by getLocationId method + there are few tests that verifies border cases (no item set, no effectiveLocatioId present)

Given this is a mixture of types of tests, please could they be split into two separate classes in the appropriate places.